test/suites/drift/suite_test.go

```diff
@@ -270,7 +277,7 @@ var _ = Describe("Drift", Label("AWS"), func() {
 		env.ExpectUpdated(pod)
 		env.EventuallyExpectNotFound(pod, node)
 	})
-	Describe("Provisioner Drift", func() {
+	Describe("One-way Drift", func() {
```

Comment:
Any reason we changed this to `One-way` drift? Isn't this just `Static Drift`?

Reply:
Node Requirement is added to Provisioner Drift section to avoid duplicating code. So I think `One-way` would be a more appropriate name

Reply:
Does it event make sense to put this into a bucket like this now that it's combining static and dynamic? Maybe just use the `DescribeTable` sections as the buckets?